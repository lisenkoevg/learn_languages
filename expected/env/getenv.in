#env ENV_VAR=PATH/WITH_NO_SPACES/WITHIN ENV_VAR2=123
#args arg1 arg2 arg3
