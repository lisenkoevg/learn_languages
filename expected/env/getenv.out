PATH/WITH_NO_SPACES/WITHIN
